api/src/routing.js

```diff
@@ -156,19 +159,41 @@ app.get('/', function(req, res) {
     // couchdb request - let it go
     proxy.web(req, res);
   } else {
-    // redirect to the app path - redirect to _rewrite
-    res.redirect(appPrefix);
+    res.sendFile(path.join(STATIC_RESOURCE_DESTINATION, 'templates/inbox.html'));
   }
 });
 
-/*
-To facilitate service worker prefetch on Chrome <66, serve a version of the app which does not require authentication
-*/
-app.get(appPrefix, (req, res, next) => {
-  if ('_sw-precache' in req.query) {
-    return res.sendFile(path.join(__dirname, 'extracted-resources/templates/inbox.html'));
+app.get(appPrefix, (req, res) => {
+  if (('deviceID' in req.query) || req.is('application/json')) {
+    // couchdb request - let it go
+    return proxy.web(req, res);
+  }
+  const parsed = url.parse(req.url);
+  parsed.pathname = '/';
+  res.redirect(url.format(parsed));
+});
+
+app.all('/medic/*', (req, res, next) => {
```

Comment:
Sounds good to me, @garethbowen any input on this?

Reply:
The problem with changing the name is when upgrading pouchdb would create a new local db and replicate everything down again assuming it's a new db.

We could solve this by intercepting the pouchdb fetch and prefixing it with whatever we want, but that makes everything really complicated, because we start off asking for `medic`, fetch changes it to `db`, then api requests it from `actual-db-name`.

I'm fairly sure that only developers will call the db anything other than "medic" so I think it should be ok.

Reply:
Well spelt out, makes sense

Reply:
Makes sense. No change then.